src/main/java/org/folio/edge/oaipmh/utils/OaiPmhOkapiClient.java

```diff
@@ -50,36 +49,17 @@ protected OaiPmhOkapiClient(Vertx vertx, String okapiURL, String tenant, long ti
    * @param headers multimap of HTTP GET headers
    * @return future with response body
    */
-  public CompletableFuture<String> call(MultiMap parameters, MultiMap headers) {
-    VertxCompletableFuture<String> future = new VertxCompletableFuture<>(vertx);
+  public void call(MultiMap parameters, MultiMap headers,
+    Handler<HttpClientResponse> responseHandler,
+    Handler<Throwable> exceptionHandler) {
     String url = getUrlByVerb(parameters);
+    headers.remove("Content-Length");
```

Comment:
Minor 2 notes...
I would move the header name to Constants
Add small comment explaining why it needs to be removed

Reply:
@epam-pavelk,
done.
Thanks.